apps/src/codebridge/hooks/useSource.ts

```diff
@@ -62,7 +63,7 @@ export const useSource = (defaultSources: ProjectSources) => {
     [setSourceHelper]
   );
 
-  const getStartSource = useCallback(() => {
+  const startSource = useMemo(() => {
```

Comment:
This didn't actually need to be a function that'd be called and return a new object (AFAIK), so I turned it into a useMemo to keep the ref stable.